src/base-components/card-base.js

```diff
@@ -6,19 +6,19 @@ import Layout from '../components/layout'
 
 
 const CardBase = ({ classes, header, content, footer, onClose }) => (
-  <div className={`inline-flex flex-col border ${classes.root}`}>
-    {onClose && <Close size='md' className={`self-end cursor-pointer ${classes.closeIcon}`} onClick={onClose} />}
+  <div className={`${classes.root || ''} inline-flex flex-col border`}>
```

Comment:
Currently, all classes keys have at least the class names in the string, so they are not empty strings. So, I would delete the ` || ''` part for all classes key values.